package/MDAnalysis/analysis/rdf.py

```diff
@@ -342,14 +342,11 @@ def _conclude(self):
 
         for i, (ag1, ag2) in enumerate(self.ags):
             # Number of each selection
-            nA = len(ag1)
-            nB = len(ag2)
-            N = nA * nB
             indices.append([ag1.indices, ag2.indices])
 
             # Average number density
             box_vol = self.volume / self.n_frames
-            density = N / box_vol
```

Comment:
Can you quickly explain again how the normalization needs to be calculated? What's the equation?

Reply:
By equation I mean the whole equation, something like the g_ab(r) at the top https://www.mdanalysis.org/docs/documentation_pages/analysis/rdf.html

Reply:
Where should I put the equation, at the top below the g_ab(r) or in the texts for InterRDF_s?